scripts/downgrade.sh.in

```diff
@@ -92,6 +92,16 @@ launch_pre_downgrade () {
         saltenv="$SALTENV"
 }
 
+launch_post_downgrade () {
+    SALT_MASTER_CALL=(crictl exec -i "$(get_salt_container)")
+    "${SALT_MASTER_CALL[@]}" salt-run saltutil.sync_all \
+        saltenv="$SALTENV"
```

Comment:
Not sure this is needed as it's already done in `launch_downgrade`, but it does not cost much to let it here.